mkt/lookup/serializers.py

```diff
@@ -2,14 +2,22 @@
 
 from django.core.urlresolvers import reverse
 
+import mkt
 from mkt.webapps.serializers import ESAppSerializer
 
 
 class AppLookupSerializer(ESAppSerializer):
     url = serializers.SerializerMethodField('get_app_summary_url')
+    status = serializers.SerializerMethodField('get_app_status')
 
     class Meta(ESAppSerializer.Meta):
-        fields = ['id', 'url', 'app_slug', 'name']
+        fields = ['app_slug', 'id', 'name', 'status', 'url']
 
     def get_app_summary_url(self, obj):
         return reverse('lookup.app_summary', args=[obj.id])
+
+    def get_app_status(self, obj):
+        status = mkt.STATUS_CHOICES_API_v2[obj.status]
+        if obj.status != mkt.STATUS_DISABLED and obj._is_disabled:
```

Comment:
This is there b/c we only store `is_disabled` in elasticsearch, which comes from a method on the app object and is true when it is either `STATUS_DISABLED` or `disabled_by_user`. So this conditional is needed to pull it back apart and only return `'disabled'` if it's `disabled_by_user`.